### PR TITLE
Epic: always use targeted article count

### PR DESCRIPTION
**Pull request by tomrf1**
The API returns two article counts for the epic:
1. the total count over 52 weeks
2. the targeted count, e.g. you can target an epic at users with 50 or more articles about climate crisis in the last 2 months

Currently the "separate article count" feature, at the top of the epic, uses the first count.
This PR changes it to use the second. So e.g. if an epic is targeted at users who've read 50 or more environment articles, the "separate article count" will display the environment article count.

In storybook, if we set the two counts to be different, we can see the targeted count is now used -
Screenshot 2024-06-07 at 08 28 30

### CODE REVIEW

File: dotcom-rendering/src/components/marketing/epics/ContributionsEpic.tsx [hunk 1]
```diff
@@ -405,14 +405,13 @@ const ContributionsEpic: ReactComponent<EpicProps> = ({
...
 					<ContributionsEpicArticleCountAboveWithOptOut
-						articleCounts={articleCounts}
+						articleCount={articleCounts.forTargetedWeeks}
 						isArticleCountOn={!hasOptedOut}
 						onArticleCountOptOut={onArticleCountOptOut}
 						onArticleCountOptIn={onArticleCountOptIn}
 						openCmp={openCmp}
 						submitComponentEvent={submitComponentEvent}
 						copy={variant.separateArticleCount?.copy}
-						countType={variant.separateArticleCount?.countType}
```
Comment (tomrf1) [inline dotcom-rendering/src/components/marketing/epics/ContributionsEpic.tsx:415]: this `countType` isn't configured anywhere, so we always end up defaulting to the `for52Weeks` count

Comment (tomrf1) [reply dotcom-rendering/src/components/marketing/epics/ContributionsEpic.tsx:415]: the alternative is to actually make `countType` configurable from the RRCP, but I don't think that's necessary currently.

**Review by tjmw — APPROVED** [review]
Looks good from an implementation perspective.

But I'm confused about the nature of the change. Will this mean users potentially see different article counts depending on the epic they're viewing? And if so is that expected? I guess it depends on the copy used but as a reader I think I'd expect `you've read X articles in the last year` to reflect the total count regardless of targeting.

**Comment by tomrf1** [comment]
> Looks good from an implementation perspective.
> 
> But I'm confused about the nature of the change. Will this mean users potentially see different article counts depending on the epic they're viewing? And if so is that expected? I guess it depends on the copy used but as a reader I think I'd expect `you've read X articles in the last year` to reflect the total count regardless of targeting.

Good question.
This was prompted by a new epic targeting users with a high article count specifically on environment content. So Marketing set up a test where the "separate article count" had the following copy:
climate_count
But the article count here was actually the total count, not just environment.

**Comment by tjmw** [comment]
> This was prompted by a new epic targeting users with a high article count specifically on environment content.

Ah okay cool thanks, yeah agreed it makes sense in that context. Eventually is it maybe something that should be configurable through the RRCP i.e. a toggle for the type of article count?

File: dotcom-rendering/src/components/marketing/epics/ContributionsEpic.tsx (hunk 1 above)
Comment (tjmw) [reply dotcom-rendering/src/components/marketing/epics/ContributionsEpic.tsx:415]: Ha I missed this when asking this same question below :-)

**Pull request merged**